pointerevents/extension/pointerevent_constructor.html

```diff
@@ -17,36 +17,79 @@
+                }, "event.getCoalescedEvents().length");
+                test(function() {
+                    assert_equals(event.target, target0);
+                }, "event.target");
+                test(function() {
+                    assert_equals(event.currentTarget, target0);
+                }, "event.currentTarget");
+                test(function() {
+                    assert_equals(event.eventPhase, Event.AT_TARGET);
+                }, "event.eventPhase");
+                test(function() {
+                    assert_equals(event.clientX, 310);
+                }, "event.clientX");
+
+                for (var i = 0; i < 2; i++) {
+                    // these properties match the dispatched event
+                    ["pointerId", "pointerType", "isPrimary", "isTrusted", "target"].forEach(
+                        function(prop) {
+                            test(function() {
+                                var coalescedEvent = event.getCoalescedEvents()[i];
+                                assert_equals(coalescedEvent[prop], event[prop]);
+                            }, "getCoalescedEvents()[" + i + "]." + prop);
+                        });
+
+                    // these are always false
+                    ["cancelable", "bubbles"].forEach(
+                        function(prop) {
+                            test(function() {
+                                var coalescedEvent = event.getCoalescedEvents()[i];
+                                assert_equals(coalescedEvent[prop], false);
+                            }, "getCoalescedEvents()[" + i + "]." + prop);
+                        });
+
+                    // these follow the defaults
+                    test(function() {
+                        var coalescedEvent = event.getCoalescedEvents()[i];
+                        assert_equals(coalescedEvent.currentTarget, null);
+                    }, "getCoalescedEvents()[" + i + "].currentTarget");
+                    test(function() {
+                        var coalescedEvent = event.getCoalescedEvents()[i];
+                        assert_equals(coalescedEvent.eventPhase, Event.NONE);
+                    }, "getCoalescedEvents()[" + i + "].eventPhase");
+
+                    // CSSOM View stuff is computed accurately for coalesced events
+                    test(function() {
+                        var coalescedEvent = event.getCoalescedEvents()[i];
+                        assert_equals(coalescedEvent.offsetX, event.offsetX + (i==0?-10:0));
+                    }, "getCoalescedEvents()[" + i + "].offsetX");
+                    test(function() {
+                        var coalescedEvent = event.getCoalescedEvents()[i];
+                        assert_equals(coalescedEvent.offsetY, event.offsetY);
+                    }, "getCoalescedEvents()[" + i + "].offsetY");
+
+                    // FIXME: gsnedders doesn't see what guarantees this but
+                    // NavidZ had this above before
```

Comment:
That is correct that the spec hasn't talked about clientX/Y at all. This was just one example of the event properties that we wanted to make sure it is set correctly (the same as what it is passed in the dictionary). It can be argued that we should have done this with all the properties and we just tested the clientX/Y as an example here.